zeppelin-web-angular/package.json

```diff
@@ -78,28 +80,31 @@
     "@types/node": "~12.19.16",
     "@types/parse5": "^5.0.2",
     "@types/webpack-env": "^1.18.8",
-    "codelyzer": "^5.0.0",
+    "@typescript-eslint/eslint-plugin": "5.27.1",
+    "@typescript-eslint/parser": "5.27.1",
     "cross-env": "^10.1.0",
-    "dotenv": "^8.0.0",
+    "dotenv": "^17.2.3",
+    "eslint": "^8.17.0",
+    "eslint-config-prettier": "^10.1.8",
+    "eslint-plugin-import": "^2.32.0",
+    "eslint-plugin-jsdoc": "^50.8.0",
+    "eslint-plugin-prefer-arrow": "^1.2.3",
     "https-proxy-agent": "^2.2.1",
     "husky": "9.1.7",
-    "lint-staged": "^8.1.6",
+    "lint-staged": "^15.5.2",
     "monaco-editor-webpack-plugin": "6.0.0",
-    "ng-packagr": "^9.1.5",
-    "ngx-build-plus": "^9.0.6",
-    "nz-tslint-rules": "^0.800.2",
-    "prettier": "^1.17.0",
-    "protractor": "~5.4.0",
+    "ng-packagr": "^13.3.1",
+    "ngx-build-plus": "^13.0.1",
+    "nz-tslint-rules": "^0.901.0",
+    "prettier": "^3.6.2",
     "scandirectory": "8.1.1",
     "ts-node": "~7.0.0",
-    "tsickle": "0.38.1",
-    "tslint": "~5.15.0",
     "tslint-no-circular-imports": "^0.7.0",
```

Comment:
If we’re removing TSLint, I think we can also remove `nz-tslint-rules` and `tslint-no-circular-imports`.
For any custom rules, we can either add them as ESLint rules if possible, or create an issue for them if not.

After that, you can remove the `tslint-rule` folder and the postinstall build step (`npm run build:tslint-rules`), and update the `lint` and `lint:fix` scripts to use ESLint instead.

It seems that `src/tslint.json` should be removed as well.